website/docs/cloud-docs/users-teams-organizations/organizations.mdx

```diff
@@ -212,6 +212,24 @@ Organization owners can set up an SSO provider for the organization.
 
 #### General
 
+Organization owners can choose between Non-Aggregated (Default) and Aggregated status checks. This setting determines whether detailed information and links are accessed directly from the VCS Provider or Terraform Cloud.
+It also determines the quantity of status checks directly sent to the VCS Provider in response to actions such as Pull/Merge requests.
+
+##### Aggregated Status Checks
+
+Aggregated status checks offer a streamlined experience to customers with "monorepo" repositories, containing configuration for a large number of workspaces.
+
+When enabled, Terraform Cloud will send one VCS status check for all runs triggered by a VCS event. If multiple workspaces are tied to a shared repository, status checks for these workspaces will be aggregated into one summary. This summary is unique to the workspace's organization and VCS client connection.
+
+The aggregated status check will allow users to access additional information in Terraform Cloud by accessing the provided "details" link. This link navigates users to a page in Terraform Cloud that offers a consolidated presentation across multiple Workspaces, highlighting details such as resource changes and possible issues that require user's attention.
+
+##### Non-aggregated Status Checks
+
+Default behavior where the VCS provider receives a status for each triggered Workspace and related Run stages (e.g. Plan, Policy Checks, Cost Estimation, Run Tasks, etc.) in response to a VCS event.
```

Comment:
I tried spreading this out a bit for clarity. Please feel free to tweak that example if it's not accurate 😅 
```suggestion
The default behavior of VCS providers is that a provider receives a status for each triggered workspace and related run stage in response to a VCS event. For example, a VCS push triggers checks for each related workspace's run stages, including the plan operation, policy checks, cost estimation, run tasks, and more. 
```